src/general/MusicCategory.cs

```diff
@@ -134,6 +155,12 @@ public class Track
 
         public string ResourcePath { get; set; } = null!;
 
+        public MusicContext[]? ExclusiveToContexts { get; set; }
```

Comment:
I think these 3 variables really need comments on them to explain this very important part of the jukebox.